### DISCUSSION TITLE
Notification for Date column [May 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2158] [Discord Threads]
> 🙋‍♂️ **ofir**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Do you wish to trigger webhook on matching a date criteria?  
#  
> 🙋‍♂️ **ofir**  
#### yes  
#  
> 🙋‍♂️ **ofir**  
#### but i want notification if date equal to "now"  
#  
> 👨‍🦲 **dstala**  
#### Right now, trigger for webhook are insert/ delete/ update record. so its not possible to configure a notification with date === now  
#  
> 👨‍🦲 **dstala**  
#### You can add a formula column as in the image & check for Formula column value in webhook  
![image0](https://cdn.discordapp.com/attachments/973458332796985376/973822434182590494/Screenshot_2022-05-11_at_11.09.26_AM.png)  
#  
> 👨‍🦲 **dstala**  
#### web-hook configuration  
![image0](https://cdn.discordapp.com/attachments/973458332796985376/973822628563394560/Screenshot_2022-05-11_at_11.13.03_AM.png)  
#  
> 👨‍🦲 **dstala**  
#### this additional condition works, but outer trigger still will be on record insert/ delete/ update.  
#  
> 🙋‍♂️ **ofir**  
#### how i create reminders for tasks?  
#  
> 👨‍🦲 **dstala**  
#### not supported at the moment  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/973458332796985376)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
